CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   - These endpoints are deprecated as license files are not required to run the platform
 - `firstNodeOccurrence` field of `storage.Node` object, which is in the response of Node endpoints, has been removed.
 - `vulns` fields of `storage.Node` object, which is in the response payload of `v1/nodes` is deprecated and will be removed in 72.0.
```

Comment:
i know it was last PR, but I think we should probably just say removed in the future unless we can orient them without postgres?

Reply:
Makes sense. It is better to not bind postgres related deprecation to a release RN.